snippets/invertKeyValues.md

```diff
@@ -0,0 +1,13 @@
+### invertKeyValues
+
+Inverts the key-value pairs of an object, without mutating it.
+
+Use `Object.keys()` and `Array.reduce()` to invert the key-value pairs of an object.
+
+```js
+const invertKeyValues = obj => Object.keys(obj).reduce((acc,key) => { acc[obj[key]] = key; return acc;},{});
```

Comment:
Should this return a Map so that any value can be a key, not just simple objects?